app/components/plans/newplan.js

```diff
@@ -1,143 +1,206 @@
+      time_per_exercise: this.state.treatment.time_per_exercise,
+      reps: this.state.treatment.reps,
+      sets: this.state.treatment.sets,
+      resistance: this.state.treatment.resistance,
+      notes: this.state.treatment.notes,
+      exercise_id: this.state.selectedExercise.id,
+      patient_id: this.props.currentPatient.id
+    };
 
-// ========Exercise Drop Down ========
-      let exercises = [];
-      fakeExerciseArray.map((exercise) => {
-        exercises.push(<MenuItem key={exercise.id} value={exercise.id} primaryText={exercise.title}/>
-                      );
-      });
-// =======Temporary Styles ===========
-      const style = {
-          marginRight: 20,
-      };
-//====================================
-  
-    const { currentpatient} = this.props;
+    if (this.state.treatments.length === 0) {
+      this.setState({ treatments: [treatment] });
+    } else {
+      let newTreatmentArray = this.state.treatments.concat(treatment);
+      this.setState({ treatments : newTreatmentArray});
+    }
+    this.setState({ treatment: initialTreatment });
+    this.setState({ selectedExercise: { title:" ", description: " "} });
+
+  }
+
+// ======= removeTreatment =======
+  removeTreatment(idx) {
+    console.log(idx);
+    let newTreatmentArray = this.state.treatments.filter((treatment,index)=> {
+      if (idx !== index) return treatment;
+    });
+    console.log(newTreatmentArray);
+    this.setState({ treatments: newTreatmentArray});
+  }
+
+//===== SubmitHandler for Entire Plan ======
+  submitHandler(evt) {
+    evt.preventDefault();
+    let newPlan = {
+      duration : this.state.duration,
+      therapy_focus : this.state.therapy_focus,
+      notes : this.state.notes,
+      patient_id: this.props.currentPatient.id,
+      treatments: this.state.treatments
+    };
+
+    console.log(`submitHandler ${newPlan}`);
+    this.props.addPlan(newPlan);
+  }
+
+
+  render(){
 
-    return(
 
-      <div className='container' id='new'>
-        <div className='row'>
-
-          <div className='col-md-8'>
-            <div className='row' id='plan-options'>
-              <div className='form' >
-                  <DropDownMenu
-                  value={this.state.duration}
-                  onChange={this.handleChange}
-                  >
-                    <MenuItem value={1} primaryText=" 1 Week"/>
-                    <MenuItem value={2} primaryText=" 2 Week"/>
-                    <MenuItem value={3} primaryText=" 3 Week"/>
-                    <MenuItem value={4} primaryText=" 4 Week"/>
-                    <MenuItem value={5} primaryText=" 5 Week"/>
-                    <MenuItem value={6} primaryText=" 6 Week"/>
-                  </DropDownMenu>
-                </div>
-
-                <div>
-                  <SelectField>
-                    <MenuItem value={1} label='Injury' />
+  // === IF Therapist has no exercises displays text instead of dropdown
+    let ExerciseList = (this.props.exercises===[]) ?
```

Comment:
This needs modification. You're saying here, if 'exercises' is an array, then display the text. Should be (this.props.exercises.length === 0) ? <display text> : <display div>